docs/api/covidcast_signals.md

```diff
@@ -182,6 +171,13 @@ amended by health authorities, this can sometimes result in negative incidence.
 This should be interpreted purely as an artifact of data reporting and
 correction.
 
+Smoothed versions of all the signals above are available with the prefix
+`7dav_`, such as `7dav_confirmed_incidence_prop` and
+`7dav_deaths_incidence_num`. These signals use report moving averages of the
+preceding 7 days, so e.g. the signal for June 7 is the average of the underlying
+data for June 1 through 7, inclusive.
```

Comment:
Could someone check that this claim is true? I didn't actually read how the smoother works

Reply:
```
temp = np.append(np.nan*np.ones(k-1), x)
    y = np.convolve(temp, np.ones(k, dtype=int), 'valid') / k
```

with k=7. So modulo an edge effect at the beginning of time, that description is correct.